schemas/jsonschema/Grid.json

```diff
@@ -112,7 +112,7 @@
       "items": {
         "$comment": "Generated from PackedElementField by @foxglove/message-schemas",
         "title": "PackedElementField",
-        "description": "List of fields for every element in a byte array",
+        "description": "Defines a field within packed data. Commonly used to represent a field of a single element within a byte array containing multiple packed elements.",
```

Comment:
Can remove "Defines" from beginning of every version of this schema. Don't think the other schemas are formatted this way.

Reply:
```suggestion
  "description": "A field present within each element in a byte array of packed elements.",
```

Reply:
Or maybe even "each packed element in a byte array".